.goreleaser.yml

```diff
@@ -0,0 +1,82 @@
+# .goreleaser.yaml
+project_name: cdk-data-availability
+
+release:
+  disable: false
+  draft: true
+  prerelease: auto
+
+builds:
+  - main: ./cmd/
+    binary: cdk-data-availability
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w
+      - -X github.com/0xPolygon/cdk-data-availability.Version={{ .Version }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitRev={{ .Commit }}
+      - -X github.com/0xPolygon/cdk-data-availability.BuildDate={{ .Date }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitBranch={{ .Branch }}
+
+archives:
+  - files:
+      - LICENSE
+      - README.md
+
+dockers:
+  - image_templates:
+      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
+    dockerfile: Dockerfile.release
+    use: buildx
+    goos: linux
+    goarch: amd64
+    ids:
+      - linux-amd64
```

Comment:
I think this should be `cdk-data-availability`

Reply:
Should be fine now https://github.com/0xPolygon/cdk-data-availability/pull/57/commits/dd5f4e48d947e2dbd91ff972eb83b9ed24346a68.

Apparently, it is working without ids as well:
https://github.com/0xPolygon/cdk-data-availability/actions/runs/7957895445/job/21721629401?pr=57